### PR TITLE
doc: the namespace is wrong when validating cilium on Azure CNI

### LINKED ISSUE #9710: doc: the namespace is wrong when validating cilium installation with Azure CNI
In https://docs.cilium.io/en/v1.6/gettingstarted/cni-chaining-azure-cni/#validate-the-installation, the namespace should be `cilium` instead of `kube-system`, because the `chaining.yaml` and the cilium release are in the namespace `cilium`.

https://docs.cilium.io/en/v1.6/gettingstarted/cni-chaining-aws-cni/ shares the validation step, whose namespace is `kube-system`.

### PR DESCRIPTION
**Pull request by soumynathan**
The namespace mentioned in the chaining.yaml is different from
the namespace queried using kubectl under validation section.
This seems to be an issue because we are re-using the file
k8s-install-validate.rst.

Fixes: #9710
Signed-off-by: Swaminathan Vasudevan <svasudevan@suse.com>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/9717)

### CODE REVIEW

File: Documentation/gettingstarted/k8s-install-azure-cni-validate.rst
Comment (bingosummer) [inline Documentation/gettingstarted/k8s-install-azure-cni-validate.rst]: Remove the lines of coredns because coredns doesn't exist in the namespace cilium.

Comment (soumynathan) [reply Documentation/gettingstarted/k8s-install-azure-cni-validate.rst]: Sure will address your comments in the next patch.

Comment (soumynathan) [reply Documentation/gettingstarted/k8s-install-azure-cni-validate.rst]: @bingosummer Fixed review comments.

**Review by bingosummer — APPROVED** [review]

**Comment by joestringer** [comment]
test-docs-please

**Review by joestringer — APPROVED** [review]
LGTM. I've kicked off the CI docs checker and I will follow up with core devs on why the maintainer's little helper is trying to block the merge of this PR. We should be able to resolve that shortly.

**Review by aanm — APPROVED** [review]

**Comment by joestringer** [comment]
Given known problems with forks and maintainers-little-helper, I will override and merge.

@soumynathan Just as a matter of style for future reference, I find it useful to phrase PR descriptions (and commit descriptions) in the imperative form: "Fix ...." ; "Refactor ..."; "Add support for ...". This way, when reading through either release notes or git one-liners, it is easy to tell the transition that this change to the code makes from one state to another, rather than declaring the state of the code either before or after and leaving it up to the reader to determine which is the case.

If we have a large set of release notes that somewhere in the middle says "the namespace is wrong ..." then it sounds like the release notes are declaring that the namespace is wrong, and we haven't fixed that yet; If the imperative "Fix the namespace in Azure CNI docs", then in the release notes it is clear that this is the release that addresses this problem.

**Comment by soumynathan** [comment]
> Given known problems with forks and maintainers-little-helper, I will override and merge.
> 
> @soumynathan Just as a matter of style for future reference, I find it useful to phrase PR descriptions (and commit descriptions) in the imperative form: "Fix ...." ; "Refactor ..."; "Add support for ...". This way, when reading through either release notes or git one-liners, it is easy to tell the transition that this change to the code makes from one state to another, rather than declaring the state of the code either before or after and leaving it up to the reader to determine which is the case.
> 
> If we have a large set of release notes that somewhere in the middle says "the namespace is wrong ..." then it sounds like the release notes are declaring that the namespace is wrong, and we haven't fixed that yet; If the imperative "Fix the namespace in Azure CNI docs", then in the release notes it is clear that this is the release that addresses this problem.

Thanks for the info. I thought about it and then I also had a dilemma if the topic should match with the bug description. Got it and will use it. Thanks for your valuable inputs.

**Pull request merged**
